.editorconfig

```diff
@@ -201,4 +201,4 @@ end_of_line = crlf
 
 [src/**/*.{cs,vb}]
 # IDE0005: Remove unnecessary usings/imports
-dotnet_diagnostic.IDE0005.severity = warning
+dotnet_diagnostic.IDE0005.severity = error
```

Comment:
I think leaving this as "warning" would make this an error only for PR and official builds.

"Unnecessary"-type errors can be a real pain while developing, because the dev might have just commented out the thing that needs the new using, in order to test an inner-loop thing. I prefer leaving them as warnings and letting them be caught by warnings-as-errors in PR builds if they didn't get cleaned up before submit.